docs/modules/ROOT/pages/mq_services/message_queue_system.html

```diff
@@ -526,16 +525,27 @@ <h3 id="_architecture">Architecture</h3>
 </div>
 <div class="sect3">
 <h4 id="_design_option_mq_node_group_topic">Design Option: MQ (Node Group Topic)</h4>
+<div class="sect4">
+<h5 id="_workflow">Workflow</h5>
 <div class="imageblock">
 <div class="content">
 <img src="../../images/file.png" alt="file">
 </div>
 </div>
 </div>
+<div class="sect4">
+<h5 id="_data_schema_for_design_option_mq_node_group_topic">Data Schema for Design Option: MQ (Node Group Topic)</h5>
+<div class="imageblock">
+<div class="content">
+<img src="../../images/dataSchema1.png" alt="dataSchema1">
+</div>
+</div>
+</div>
+</div>
 <div class="sect3">
 <h4 id="_design_option_mq_vpc_topic_grpc">Design Option: MQ (VPC Topic) + GRPC</h4>
 <div class="sect4">
-<h5 id="_workflow">Workflow</h5>
+<h5 id="_workflow_2">Workflow</h5>
 <div class="imageblock">
 <div class="content">
 <img src="../../images/vpcGroup.png" alt="vpcGroup">
```

Comment:
Could you check the image name under /image folder? Should be vpcGroup.png, not vpcGrouppng. Thanks.

Reply:
@issacyxw Where is the image vpcGroup.png?

Reply:
vpcGroup.png exists in the  /image folder